Sources/Core/ModernAVPlayer.swift

```diff
@@ -110,6 +111,16 @@ public final class ModernAVPlayer: NSObject, MediaPlayer {
         }
         context.loadMedia(media: media, autostart: autostart)
     }
+   
+    /**
+     Replaces the current player metadata with the new metadata
+     This function is dedicated for updating live data, and though, will discard duration and isLive previously set.
+     
+     - parameter metadata: metadata to load
+     */
+    public func updateLive(metadata: PlayerMetadata) {
```

Comment:
I suggest renaming method to -> updateNowPlayingInfo(metadata: PlayerMetadata)

Reply:
ok